src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java

```diff
@@ -109,6 +116,8 @@ public UnlinkedFilesDialogView() {
     private void initialize() {
         viewModel = new UnlinkedFilesDialogViewModel(dialogService, undoManager, fileUpdateMonitor, preferencesService, stateManager, taskExecutor, importFormatReader);
 
+        this.bibDatabaseContext = stateManager.getActiveDatabase().orElseThrow(() -> new NullPointerException("Database null"));
```

Comment:
Can you improve the error message please? Remember we are dealing with users, not coders. ;-)

Reply:
Please also check that the calling UI command requires an opened database.

Reply:
Already checked in the action